cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsInstanceConnector.java

```diff
@@ -134,18 +134,21 @@ public String getConsoleOutput(AuthenticatedContext authenticatedContext, CloudI
         AmazonEC2Client amazonEC2Client = awsClient.createAccess(new AwsCredentialView(ac.getCloudCredential()),
```

Comment:
this is not used anymore

Reply:
Thank you for pointing this out. It is removed.